test/src/libc-test/functional/argv.c

```diff
@@ -0,0 +1,2 @@
+//! add-flags.py(CFLAGS): -I.
+#include "build/download/libc-test/src/functional/argv.c"
```

Comment:
Should we add a comment to all these that says something like `Auto-generated by genrate-stubs.sh`?

Reply:
Good idea, added.